stories/Organism/Articlepage/ArticlePage.jsx

```diff
@@ -0,0 +1,142 @@
+import React, { useEffect } from 'react';
+import './ArticlePage.scss';
+import { PostHeader } from '../Text/PostHeader/PostHeader';
+import { Images } from '../Images/Images/Images';
+import { Imagecaption } from '../../Molecules/Imagecaption/Imagecaption';
+import { Author } from '../../Molecules/Text/Authors/Authorcolumn/Authors';
+import { BlockquoteComponent } from '../../Molecules/Text/BlockquoteComponent/BlockquoteComponent';
+import { P } from '../../Atom/Base-typography/Paragraph/Paragraph';
+import { List } from '../../Atom/Lists/Lists';
+import { FeaturedCard } from "../Blocks/FeaturedContentCard/FeaturedCard/FeaturedCard";
+import { Heading } from '../../Atom/Base-typography/Heading/Heading';
+import farmlandlg from '../../assets/images/farmland-lg.jpg';
+import farmlandmd from '../../assets/images/farmland-md.jpg';
+import farmlandsm from '../../assets/images/farmland-sm.jpg';
+import user from '../../assets/images/user.svg';
+import { Footer } from '../Footer/Footer';
+import logo from '../../assets/images/logo.svg';
+import logowhite from '../../assets/images/logo-white.svg';
+import viewport from '../../assets/js/viewport';
+
+const ArticlePage = ({
+  data, header, country, description,
+  imageCaptionData, authorCardData, blockquoteData, paraLabel,
+  headingData, listData, footerData, headingLabel, featuredCardContent,
+}) => {
+  useEffect(() => {
+    viewport('.feature__card--headertext');
+  }, []);
+  return (
+    <div>
```

Comment:
@Ashish-srij This div tag is not required instead of that we can add empty fragment like this <></>